compiler/src/codegen/compcore.re

```diff
@@ -2119,6 +2119,102 @@ let allocate_record = (wasm_mod, env, ttag, elts) => {
   );
 };
 
+// TODO mayb
+let allocate_uint_uninitialized =
+    (~additional_words=?, wasm_mod, env, is_32_bit) => {
```

Comment:
On second inspection it doesn't actually look like this function will ever be called (no grain code ever calls `allocateUint32` or `allocateUint64`). Should I just remove this function or leave it in in case it's needed sometime in the future?

Reply:
I would leave it.